src/sgame/sg_spawn_mover.cpp

```diff
@@ -2514,6 +2514,11 @@ void SP_func_bobbing( gentity_t *self )
 	VectorCopy( self->s.origin, self->s.pos.trBase );
 	VectorCopy( self->s.origin, self->r.currentOrigin );
 
+	if ( self->config.speed == 0 )
```

Comment:
the default value should be 4 if not set (says netradiant)
So the question is why its 0 and where the default settings are set.

A bit above damage seems to be initiated, that could be the method needed here.
```
G_ResetIntField(&self->damage, true, self->config.damage, self->eclass->config.damage, 2);
```

Reply:
If I do instead:

```c++
G_ResetFloatField(&self->speed, true, self->config.speed, self->eclass->config.speed, 4);
```

It fixes nothing.